src/main/java/com/powsybl/openloadflow/sensi/ComputedElement.java

```diff
@@ -0,0 +1,131 @@
+/**
```

Comment:
You are adding new classes that are not specific to sensi at all (since you developper them for actions on SA)
Instead of adding more classes to the sensi package, why not moving everything that is specific to Woodbury to a new package com.powsybl.openloadflow.dc.fastdc ?

Reply:
I agree 100%. In order to simplify the review of the modifications of this PR (as many elements specific to Woodbury are modified), I propose to move the code in another PR. I open it ASAP.

Reply:
Done in #1145.

Reply:
Merged. I resolve the conversation.